translate/storage/fpo.py

```diff
@@ -329,10 +328,6 @@ def hasplural(self):
         source = self.source
         return isinstance(source, multistring) and len(source.strings) > 1
 
-    def parse(self, src):
-        raise DeprecationWarning("Should not be parsing with a unit")
```

Comment:
In https://github.com/claudep/translate/blob/remove_fpo_depr/translate/storage/fpo.py#L291